src/load_dotenv.rs

```diff
@@ -1,43 +1,57 @@
 use crate::common::*;
 
-pub(crate) fn load_dotenv(
+fn load_from_file(
```

Comment:
Let's move this function after `load_dotenv` in the file, since it's always called afterwards.